quiz-app/src/components/app.js

```diff
@@ -0,0 +1,182 @@
+      <Grid container direction={"column"} spacing={2} columns={{ xs: 12, sm: 12, md: 12, lg: 12, xl: 12 }}>
+        <Grid item xs={6} align="left">
+          <p1> Enter Question 1:</p1>
+        </Grid>
+        <Grid item align="left">
+          <TextField
+            value={question}
+            placeholder="Enter Question"
+            label="Quiz question"
+            name="Quiz Question"
+            variant="outlined"
+            onChange={t => {
+              setQuestion(t.target.value)
+            }}
+          />
+        </Grid>
+        <Grid item xs={6}>
+          <FormControl component="fieldset">
+            <FormLabel component="legend" id="correct answer">Answers: </FormLabel>
+              <Grid item sx={{ py: 2 }}>
+                <Grid container direction={"row"} spacing={20}>
+                  <Grid item>
+                    <p12> Enter Choice 1:</p12>
+                  </Grid>
+                  <Grid item>
+                    <p22> Correct Answer:</p22>
+                  </Grid>
+                </Grid>
+              </Grid>
+              <Grid container direction={"row"} spacing={9} alignItems="center">
+                <Grid item>
+                  <TextField
+                    value={choice1}
+                    placeholder="Enter Choice 1"
+                    label="Choice 1"
+                    name="Choice 1"
+                    variant="outlined"
+                    onChange={t => {
+                      setChoice1(t.target.value)
+                    }}
+                  />
+                </Grid>
+                <Grid item>
+                  <FormControlLabel
+                    control={
+                      <Checkbox
+                        checked={answer1}
+                        onChange={handleAnswerCorrectnessChange}
+                        name="answer1"
+                      />
+                    }
+                  />
+                </Grid>
+              </Grid>
+
+              <Grid item sx={{ py: 2, mr: 35 }}>
+                <p2> Enter Choice 2:</p2>
+              </Grid>
+              <Grid container direction={"row"} spacing={9} alignItems="center">
+                <Grid item>
+                  <TextField
+                    value={choice2}
+                    placeholder="Enter Choice 2"
+                    label="Choice 2"
+                    name="Choice 2"
+                    variant="outlined"
+                    onChange={t => {
+                      setChoice2(t.target.value)
+                    }}
+                  />
+                </Grid>
+                <Grid item>
+                  <FormControlLabel
+                    control={
+                      <Checkbox
+                        checked={answer2}
+                        onChange={handleAnswerCorrectnessChange}
+                        name="answer2"
+                      />
+                    }
+                  />
+                </Grid>
```

Comment:
We know you will have to change it if you want a dynamic number of answers (so you won't be able to call them answer1, answer2...). Could you try to create an array which will contain the answers and dynamically create the components related to it?

```
const [answers, ...] = useState([{ value: 'it is answer 1', checked: false }])

...

answers.map((answer, index) => {
return (
    <Grid item>
                  <TextField
                    value={answer.value}
                    placeholder={answer.placeholder}
                    label={answer.label}
                    name={answer.name}
                    variant="outlined"
                    onChange={t => {
// change the value here, it's possible to get the array index from the map
                      setAnswers(...)
                    }}
                  />
                </Grid>
)
})
```